httpConn_test.go

```diff
@@ -235,7 +235,7 @@ func TestStart(t *testing.T) {
 	count := 0
 	foundNewSpan := false
 	for k, v := range resp.Header {
-		if k == HEADER {
+		if strings.ToLower(k) == strings.ToLower(HEADER) {
```

Comment:
Good catch - that will be faster and more clear.
On Jul 8, 2016 1:35 PM, "Khem Raj" <notifications@github.com> wrote:

> In httpConn_test.go
> <https://github.com/Comcast/golang-money/pull/9#discussion_r70136806>:
>
> > @@ -235,7 +235,7 @@ func TestStart(t *testing.T) {
> >  	count := 0
> >  	foundNewSpan := false
> >  	for k, v := range resp.Header {
> > -		if k == HEADER {
> > +		if strings.ToLower(k) == strings.ToLower(HEADER) {
>
> how about using strings.EqualFold(str1, str2)
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Comcast/golang-money/pull/9/files/6e930b19a8c348b719a962c737c983d98ac4e56b..2aa2753bdd1f403c5032ab987acdcc2fc0ae5dea#r70136806>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAqpyfZF0X5R_HIkgecpqnbpCn4Yx0mDks5qTrR5gaJpZM4JFbDz>
> .
>